react/index.d.ts

```diff
@@ -23,7 +27,10 @@ import { ReadableStore } from '../create-store/index.js'
  * @param store Store instance.
  * @returns Store value.
  */
-export function useStore<Value extends any>(store: ReadableStore<Value>): Value
+export function useStore<Value extends any>(
+  store: ReadableStore<Value>,
+  options?: UseStoreOptions
+): Value
```

Comment:
On `only` option, we need to modify returning `Value` to avoid using keys which was not `only` options

Reply:
We have no filter inside the hook, we only mute events...
Should we filter return value as well?

Reply:
I'm afraid that somebody will forget to update `keys` on changing the React template and then will spend a day debugging React render. We do not need to change returning value, just change type to help user